docs/install.rst

```diff
@@ -169,13 +169,26 @@ Name                                 Notes
                                      audio and video streams, eg. YouTube 1080p+
 ==================================== ===========================================
 
+Using pycrypto and pycountry
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
I'm surprised this didn't fail the build. Could you make the `^^` underline the same width and the text above?